tests/model_serving/model_runtime/mlserver/conftest.py

```
@@ -0,0 +1,158 @@
+    s3_models_storage_uri: str,
+    model_service_account: ServiceAccount,
+) -> Generator[InferenceService, Any, Any]:
+    isvc_kwargs = {
+        "client": admin_client,
+        "name": request.param["name"],
+        "namespace": model_namespace.name,
+        "runtime": serving_runtime.name,
+        "storage_uri": s3_models_storage_uri,
+        "model_format": serving_runtime.instance.spec.supportedModelFormats[0].name,
+        "model_service_account": model_service_account.name,
+        "deployment_mode": request.param.get("deployment_type", KServeDeploymentType.RAW_DEPLOYMENT),
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)